src/main/java/com/gooddata/warehouse/WarehouseS3Credentials.java

```diff
@@ -101,13 +109,6 @@ public DateTime getUpdated() {
     }
 
     /**
-     * @return URI of the user who updated the credentials last
-     */
-    public String getUpdatedBy() {
```

Comment:
I plan to use new `WarehouseS3Credentials.withLinks(warehouseId)` method in the converter for creating all the links, and therefore converter won't need access to `updatedBy` anymore.